### PR TITLE
API 4877 Location Identifier Update

### PR DESCRIPTION
**Pull request by that-guy-is-it**
Populate the new Location columns.

### CODE REVIEW

**Comment by that-guy-is-it** [comment]
@nmoisan @jtaylor-va Pinging as requested!

File: minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java [hunk 1]
```diff
@@ -362,6 +363,9 @@ private void insertByLocation(File file) {
             .city(dm.address().city())
             .state(dm.address().state())
             .postalCode(dm.address().postalCode())
+            .stationNumber(facilityId == null ? null : facilityId.stationNumber())
```
Comment (jblefkove-va) [inline minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java:366]: use `Optional.map()`: `dm.facilityId().map(fid -> fid.stationNumber()).orElse(null)`

File: minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java [hunk 2]
```diff
@@ -362,6 +363,9 @@ private void insertByLocation(File file) {
...
+            .facilityType(dm.type().orElse(null))
```
Comment (jblefkove-va) [inline minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java:367]: that's the type of the location, not the type of the facility

Comment (that-guy-is-it) [reply minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java:367]: Hmm... using the previous comment for reference... would the facility type be accessed as: 

`.facilityType(dm.facilityId().map(fid -> fid.type().fapiPrefix()).orElse(null))`?

Comment (jblefkove-va) [reply minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java:367]: Looking at how it's done for Organization, I think you want `type.toString()`. (You should also protect against `type()` being null.)

**Review by nmoisan — COMMENTED** [review]
Outside of Peter's comments, I have nothing else to add.

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**